src/lib.rs

```diff
@@ -107,23 +107,7 @@
 //! to allow spaces.  The first argument is the program to run which maps to `bin.name` in the
 //! `.toml` file.
 //!
-//! Example:
-//!
-//! With a `[[bin]]` like:
-//! ```rust,ignore
-//! fn main() {
-//!     println!("Hello world");
-//! }
-//! ```
-//!
-//! You can verify a code block like:
-//! ~~~md
-//! ```console
-//! $ my-cmd
-//! Hello world
-//!
-//! ```
-//! ~~~
```

Comment:
I lean towards us having both.  The inline example first and then a "for a more complete example, see ..."

Reply:
Sounds good.